packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/config.ts

```diff
@@ -10,16 +10,6 @@ export enum Runtime {
    */
   NODEJS_18_X = 'nodejs18.x',
 
-  /**
-   * The Python 3.9 runtime
-   */
-  PYTHON_3_9 = 'python3.9',
-
-  /**
-   * The Python 3.10 runtime
-   */
-  PYTHON_3_10 = 'python3.10',
-
```

Comment:
is it mandatory to remove these values ... this is an exposed Enum, and may be some customers are using these values .. do you think it is not a correct understanding

Reply:
I think you raised a good point. Although there is a better place to use the ENUM from ([here](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-lambda/lib/runtime.ts#L53)), Customers might be using these runtime enum for some unknown reason. Let me add it back.